e2e/scenario_test.go

```diff
@@ -824,18 +824,20 @@ func Test_Ubuntu2204_ArtifactStreaming_Scriptless(t *testing.T) {
 	})
 }
 
-func Test_Ubuntu2204_ChronyRestarts(t *testing.T) {
+func Test_Ubuntu2204_ChronyRestarts_Taints_And_Tolerations(t *testing.T) {
```

Comment:
Intentionally adding taints test into an existing test to save some testing cost. Otherwise it will need to spin up a new node to test which is expensive.

Reply:
I think we should do this more - ideally have one Test function per VHD and run a lot of scenarios on it.

Reply:
Ya. If the tests are not conflicting with one another.